test/unit/org/apache/cassandra/utils/CassandraGenerators.java

```diff
@@ -309,9 +311,8 @@ public static Gen<String> sstableFormatNames()
 
     public static Gen<SSTableFormat<?, ?>> sstableFormat()
     {
-        // make sure ordering is determanstic, else repeatability breaks
-        NavigableMap<String, SSTableFormat<?, ?>> formats = new TreeMap<>(DatabaseDescriptor.getSSTableFormats());
-        return SourceDSL.arbitrary().pick(new ArrayList<>(formats.values()));
+        return SourceDSL.arbitrary().pick(new BigFormat(Map.of()),
```

Comment:
this is just making sure that the config doesn't alter the random history; we only allow BIG/BTI atm and not anything randomly included